src/main/java/ezvcard/VCard.java

```diff
@@ -108,6 +113,8 @@
  * Represents a vCard.
  * @author Michael Angstadt
  */
+@JsonSerialize(using = JCardSerializer.class)
```

Comment:
Removing them is no problem. The annotations could go on the getter/setter methods of the parent object, or the ObjectMapper can be configured to use the vcard serializers. My thinking was that you would need the dependency on jackson-databind regardless, as that's where the Serializer/Deserializer base classes are defined, but now you mention it, it could just as easily be an optional dependency. Anybody who needs this functionality would probably already have the dependency on jackson-databind in the first place.